src/routes/fest-route/fest-route.jsx

```diff
@@ -25,11 +28,27 @@ export const FestRoute = () => {
     { data: updateData, loading: updateLoading, error: updateError },
   ] = useMutation(UPDATE_FEST, { refetchQueries: [GET_FEST] });
 
+  // route a user away from /fest/:festId endpoint if they are not an attendee or invitee of the fest
```

Comment:
This useEffect enables authorization for the fest/festId endpoint, so that a user who is not invited to the fest can not access this route. It may be that we want a more re-usable code for that kind of authorization?